src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
-        cbsd_ids.append(response['cbsdId'])
-      else:
-        self.assertFalse('cbsdId' in response)
-
-    # Request grant
-    config['grantRequests'][0]['cbsdId'] = cbsd_ids[0]
-    request = {'grantRequest': config['grantRequests']}
-    responses = self._sas.Grant(request)['grantResponse']
-    # Check Grant responses.
-    for i in range(len(responses)):
-      response = responses[i]
-      expected_response_codes = config['expectedGrantResponseCodes']
-      logging.debug('Expecting to see response code in set %s in response: %s',
-                      expected_response_codes, response)
-      self.assertIn(response['response']['responseCode'],
-                      expected_response_codes)
-      if response['response']['responseCode'] == 0:  # SUCCESS
-          self.assertTrue('cbsdId' in response)
-          self.assertTrue('grantId' in response)
-          grant_expire_time = datetime.strptime(response['grantExpireTime'],
-                                                '%Y-%m-%dT%H:%M:%SZ')
-      else:
-          self.assertFalse('cbsdId' in response)
-          self.assertFalse('grantId' in response)
-
-    # Check grant response
-    self.assertEqual(responses[0]['cbsdId'], cbsd_ids[0])
-    grant_id = responses[0]['grantId']
-    grant_expire_time = datetime.strptime(responses[0]['grantExpireTime'],
-                                          '%Y-%m-%dT%H:%M:%SZ')
-    del request, responses
-
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
-    request = {'grantRequest': config['grantRequests']}
-
-    # Check grant response
-    response = self._sas.Grant(request)['grantResponse'][0]
-    self.assertEqual(response['cbsdId'], cbsd_ids[0])
-    self.assertIn(response['response']['responseCode'], config['expectedGrantResponseCodes'])
-    grant_id = response['grantId']
-    grant_expire_time = datetime.strptime(response['grantExpireTime'],
-                                          '%Y-%m-%dT%H:%M:%SZ')
-    del request, response
-
-    # Load FSS database which includes at least one FSS site near the above registered CBSD location
-    self._sas_admin.InjectDatabase_url(config['fss_db'])
+            'grantId': grant_ids[0]
+          }]
+        }
+      response = self._sas.Relinquishment(request)['relinquishmentResponse'][0]
+      # Check the relinquishment response
+      self.assertEqual(response['cbsdId'], cbsd_ids[0])
+      self.assertEqual(response['grantId'], grant_ids[0])
+      self.assertIn(response['response']['responseCode'], [400])
+
+    #Request grant with frequency range which partially or fully overlaps with \
+    #Exclusion zone protected frequency range 'F1'
+    config['grantRequest'][0]['cbsdId'] = cbsd_ids[0]
+    #Get an overlapping frequency range for the Exclusion zone Frequency range(F1)
+    freq_range = getOverlappingFrequency(\
```

Comment:
This should be taken from the config file, not hard-coded into the test case.

Reply:
Defined   the overlapping frequency ranges for 'F1' and 'F2' in generate config method.